src/client/pages/Offer/utils.ts

```diff
@@ -32,3 +32,7 @@ export const getAddress = (quotes: BundledQuote[]) => {
       })
     : ''
 }
+
+export const bundleContainsCarQuote = (bundle: BundledQuote[]) => {
```

Comment:
Maybe this is not a "bundle"? I can happily change name if anyone has any suggestions.

Reply:
Maybe this can leave inside [quoteBundleSelectors](https://github.com/HedvigInsurance/web-onboarding/blob/master/src/client/api/quoteBundleSelectors.ts#L21)